### DISCUSSION TITLE
In 2024, what is the BEST template that still makes my profile README look cool and unique.

### DISCUSSION THREAD

**Discussion by Visual-Studio-Coder** [community/community#120588] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've seen ppl get super creative with their profiles, but I was wondering what the best way for a lazy guy like me was to make a cool looking profile with low effort!

**Comment by aaronhma** [comment]
I wouldn't recommend using templates, I would recommend just being yourself! Let your personality shine! :)

**Reply by Visual-Studio-Coder** [reply]
omg ur so hepful

**Comment by ettaboyle** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
